operations/tests.py

```diff
@@ -78,12 +96,33 @@ def test_api_create_trainer_detailed(self):
                 "coffee": "black"
             }
         }
+
+        # Execute
         response = self.client.post('/api/v1/trainers/',
                                     json.dumps(post_data),
                                     content_type='application/json')
+
+        # Check
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         d = Trainer.objects.last()
         self.assertEqual(d.name, "Test User 2 Detailed")
         self.assertEqual(d.msisdn, "+27820020002")
         self.assertEqual(d.email, "user2@operations.com")
         self.assertEqual(d.extras, {"id": "1234561111222", "coffee": "black"})
+
+    # Location Api Testing
+    def test_api_get_location(self):
+        # Setup
+        location_data = {
+            "point": Point(18.0000000, -33.0000000)
+        }
+        location = Location.objects.create(**location_data)
+
+        # Execute
+        response = self.client.get(
+            '/api/v1/locations/%s/' % location.id,
+            content_type='application/json')
+
+        # Check
+        self.assertEqual(response.data["point"],
```

Comment:
Here's what a JSON point helper looks like: https://github.com/praekelt/nightingale/blob/develop/reports/tests.py#L116-L126

I think it might improve this test?